Makefile

```diff
@@ -53,6 +53,10 @@ Drivers/Embedded-Base/general/src/sht30.c \
 Drivers/Embedded-Base/middleware/src/timer.c \
 Drivers/Embedded-Base/middleware/src/ringbuffer.c \
 Drivers/Embedded-Base/middleware/src/c_utils.c \
+Drivers/adbms/lib/src/adBms6830GenericType.c \
+Drivers/adbms/lib/src/adBms6830ParseCreate.c \
+Drivers/adbms/program/src/serialPrintResult.c \
+Drivers/adbms/program/src/mcuWrapper.c \

```

Comment:
should we be using this file?  Versus just defining our own.

Reply:
It has our edits in it (just the ones we used to get shep working at first)